Makefile.am

```diff
@@ -813,6 +813,20 @@ endif
 
 check: $(CHECK_DEPS)
 
+install-all: install-kernel |  install-image |  install
+
+# Get clear containers image path
+IMAGE_PATH := $(shell dirname $(CONTAINERS_IMG))
+install-image:
+	curl https://raw.githubusercontent.com/clearcontainers/tests/master/.ci/install_clear_image.sh \
```

Comment:
Well the reason I did this was to avoid the duplication of the code, and I wanted to get feedback from the devs of this project first. 

The main problem is that yes, at some point the scripts downloaded from the other repo need root to install the artifacts ``kernel`` and ``image`` needed to run clear containers.

IMHO the best option is to  adapt the scripts  to our own needs with some minnor tweaks. And add them to our repo. After all, the scripts in the other repo pursue a different goal. 

The feedback @jodh-intel gave is great but I think we are overcomplicating things with a fix that can be simple as this.